core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -667,12 +667,12 @@ private[spark] object SparkConf extends Logging {
         translation = s => s"${s.toLong * 10}s")),
     REDUCER_MAX_SIZE_IN_FLIGHT.key -> Seq(
       AlternateConfig("spark.reducer.maxMbInFlight", "1.4")),
-    "spark.kryoserializer.buffer" -> Seq(
```

Comment:
For these `spark.kryoserializer.buffer` and `spark.kryoserializer.buffer.max`, it seems there are too many instances.
```
core/src/test/scala/org/apache/spark/SparkConfSuite.scala:    assert(conf.getSizeAsKb("spark.kryoserializer.buffer") === 1100)
core/src/test/scala/org/apache/spark/storage/BlockManagerReplicationSuite.scala:  conf.set("spark.kryoserializer.buffer", "1m")
core/src/test/scala/org/apache/spark/storage/BlockManagerReplicationSuite.scala:  conf.set("spark.kryoserializer.buffer", "1m")
core/src/test/scala/org/apache/spark/storage/BlockManagerReplicationSuite.scala:  conf.set("spark.kryoserializer.buffer", "1m")
core/src/test/scala/org/apache/spark/storage/BlockManagerSuite.scala:  val serializer = new KryoSerializer(new SparkConf(false).set("spark.kryoserializer.buffer", "1m"))
core/src/test/scala/org/apache/spark/storage/BlockManagerSuite.scala:      .set("spark.kryoserializer.buffer", "1m")
core/src/test/scala/org/apache/spark/storage/MemoryStoreSuite.scala:  val serializer = new KryoSerializer(new SparkConf(false).set("spark.kryoserializer.buffer", "1m"))
examples/src/main/scala/org/apache/spark/examples/mllib/MovieLensALS.scala:        .set("spark.kryoserializer.buffer", "8m")
```

```
mllib/src/main/scala/org/apache/spark/ml/feature/Word2Vec.scala:        sc.conf.get("spark.kryoserializer.buffer.max", "64m"))
mllib/src/main/scala/org/apache/spark/mllib/feature/Word2Vec.scala:        spark.conf.get("spark.kryoserializer.buffer.max", "64m"))
```

Reply:
You can use `git grep` to search these instances, @wenxuanguan .

Reply:
@dongjoon-hyun Sorry for delay. 
I have fix the test, and changed some configs with the same problem.